lib/ASRouter.jsm

```diff
@@ -921,6 +956,27 @@ class _ASRouter {
     }
   }
 
+  async forceAttribution(data) {
```

Comment:
This is for `about:newtab#asrouter`

Reply:
can you add a comment that says that basically?